core-rust/core-api-server/src/core_api/conversions/substates/transaction_tracker.rs

```diff
@@ -12,12 +13,14 @@ pub fn to_api_transaction_tracker_substate(
     Ok(field_substate!(
```

Comment:
because of:
```
error[E0599]: no method named `as_latest_ref` found for reference `&radix_engine_queries::typed_substate_layout::TransactionTrackerSubstate` in the current scope
   --> core-api-server/src/core_api/conversions/substates/mod.rs:101:64
    |
101 |                       let $value_unpacking = $substate.payload().as_latest_ref()
    |                                                                  ^^^^^^^^^^^^^ method not found in `&TransactionTrackerSubstate`
    |
...
```

The versioned substate helpers for transaction tracker are apparently hand-crafted (i.e. not using `declare_native_blueprint_state!`). I assumed there is a reason for it (some custom handling needs, etc) - are you saying that it is simply an omission on Engine side?

Reply:
Ahh I see, weird. But okay, no worries :) - if/when it's fixed engine side we can fix it here too.